static/index.html

```diff
@@ -143,11 +143,11 @@
                 <div class="container">
                     <div class="row align-items-start">
                         <div class="col-lg-8">
-                            {%if printer_serial %}
+                            {%if printer %}
                             <div class="row mb-3">
                                 <div class="container rounded-3" id="connection">
                                     <div class="p-3 rounded-3 fs-6 bg-body-tertiary">
-                                        Connected to: <strong>{{ printer_serial }}</strong>
+                                        Connected to: <strong>{{ printer.name | e }} ({{ printer.sn }})</strong>
```

Comment:
I read up on Flask and autoescaping.

It seems that autoescaping is enabled for all templates that end in (among other extensions), `.html`, when using `render_template()`. So that covers us here, and that matches with your tests.

So we have to *not* escape, since otherwise we render some names incorrectly. For example, a printer called `Foo & Bar` would come out wrong (probably rendered as `Foo &amp; Bar`, from `Foo &amp;amp; Bar`

Reply:
✅